test/integration/federation/framework/secret.go

```diff
@@ -31,7 +31,7 @@ import (
 )
 
 type SecretAdapter struct {
-	client federationclientset.Interface
+	Client federationclientset.Interface
```

Comment:
Is there a reason to choose one approach over another?  I've generally preferred to use factory methods only when more than simple assignment is required.